docs/graphics/building-blocks/SheetMobile.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="312" height="180" fill="none"><path fill="#005C62" d=""/></svg>
```

Comment:
I'm seeing this as blank — and also the text is overflowing the card:

![Screen Shot 2023-04-03 at 5 19 25 PM](https://user-images.githubusercontent.com/12059539/229654529-7ca0d4a4-e114-4cb6-bdf6-324d5ab79cdf.png)

Reply:
Yes. I still don't have an SVG

Reply:
DONE